src/editors/vscode/scripts/bogo.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+
+
+def get_concurrency():
+    def_concurrency = 2
+    max_concurrency = 16
+
+    try:
+        import multiprocessing
+        return min(max_concurrency, multiprocessing.cpu_count())
+    except ImportError:
+        return def_concurrency
+
+
+def run_cmd(cmd, workingdir=None):
+    if isinstance(cmd, str):
+        print('> running: ' + cmd)
+        shell = True
+    else:
+        print('> running: ' + ' '.join(cmd))
+        shell = False
+    sys.stdout.flush()
+
+    try:
+        subprocess.run(cmd, shell=shell, check=True, cwd=workingdir)
+    except subprocess.CalledProcessError:
+        raise RuntimeError('External command failed, aborting...')
+
+
+boring_repo = "https://github.com/reneme/boringssl.git"
+boring_branch = "rene/runner-20220322"
```

Comment:
I guess it would be great to centralize that somewhere. The same config is duplicated in the CI setup.

Reply:
Agree that it would be good to have a central configuration. In general I would try to avoid doing similar stuff for CI and local development in different scripts. And instead try to write a general one that can be reused.

But maybe not that easy here and a little out of scope for the PR.

Reply:
Agree. Let's have a cleanup PR after this one. Also, I'm duplicating some scripts from the Sublime integration (which probably no-one uses or maintains now).